External/FEXCore/Source/Interface/Core/OpcodeDispatcher.cpp

```diff
@@ -2372,6 +2372,29 @@ void OpDispatchBuilder::RORX(OpcodeArgs) {
   StoreResult(GPRClass, Op, Result, -1);
 }
 
+void OpDispatchBuilder::MULX(OpcodeArgs) {
+  // RDX is the implied source operand in the instruction
+  const auto RDXOffset = offsetof(FEXCore::Core::CPUState, gregs[FEXCore::X86State::REG_RDX]);
+  const auto OperandSize = GetSrcSize(Op);
+
+  // Intel manual states that operands are always 32-bit
```

Comment:
I believe this is incorrect. It should be that VEX.W is ignored. So 32-bit is always 32-bit operation.
Encoding the assembly manually for a 32-bit ASM test would prove this.

Reply:
Amended to remove the decode failure

Reply:
With the decode failure removed. What is the current behaviour of VEX.W in 32-bit?
Does FEX currently debug this as a 64-bit op incorrectly?

Reply:
Not anymore =)